BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -36,6 +39,32 @@ class TipsDetailViewController: UIViewController {
     tipsView.tableView.dataSource = self
 
     title = Strings.Tips
+    loadData()
+  }
+  
+  private func loadData() {
+    _ = getTipsThisMonth().then {
+      totalBatTips = $0.total.isEmpty ? "0.0" : $0.total
+    }
+    
+    let dateFormatter = DateFormatter().then {
+      $0.dateStyle = .short
+      $0.timeStyle = .none
+    }
+    // Fixme: Check if recurreing tips happen at autoContribute reconcileStamp
+    let reconcileDate = Date(timeIntervalSince1970: TimeInterval(state.ledger.autoContributeProps.reconcileStamp))
+    nextContributionDateView.label.text = dateFormatter.string(from: reconcileDate)
+    nextContributionDateView.bounds = CGRect(origin: .zero, size: nextContributionDateView.systemLayoutSizeFitting(UIView.layoutFittingExpandedSize))
+    
+    state.ledger.listOneTimeTips {[weak self] infoList in
+      self?.tipsList.append(contentsOf: infoList)
```

Comment:
Should `guard self = self else { return }` if you're going to use weak self more than once

Reply:
I find it unnecessary, but will update :)